extendr-api/src/wrapper/mod.rs

```diff
@@ -197,6 +199,7 @@ make_conversions!(S4, ExpectedS4, is_s4, "Not a S4 type");
 make_conversions!(Integers, ExpectedInteger, is_integer, "Not an integer type");
 make_conversions!(Logicals, ExpectedLogical, is_logical, "Not a logical type");
 make_conversions!(Doubles, ExpectedReal, is_real, "Not a floating point type");
+make_conversions!(Complexes, ExpectedComplex, is_complex, "Not a complex type");
```

Comment:
Maybe 'complex number'? 'Complex type' can be misleading.

Reply:
Ok.

Reply:
Also, next step is to ditch `FromRobj`